Caiti branch ii

laikasghost opened the pull request:
Created credits scene and added button to main menu

Commit:
Created credits

Credits scene created, still needs to be attached to things

Commit:
Added credits button

Working (hopefully) credits button added to main menu

Commit:
added credits to build

Added credits scene to build settings (thought this was pushed before, oops)

Commit:
Esc credits

Esc key loads main menu while in credits scene

Commit:
moved creditescape.cs

Moved to /Util

SiennaC reviewed (changes requested):
Please make sure the credits button scales appropriately for larger screen sizes. (Anchors should be at the corners of the button - you can use the uGUI menu to do this easily.)

Also could you please put the credits button before quit? And check to make sure spacing is even between all buttons (they should be lined up exactly to the edges of the next/previous button) & that all anchors are correct (when you move a button, you have to reposition its anchors).

Commit:
Fixed menu buttons

Fixed position of anchors and order of menu buttons.

SiennaC reviewed (changes requested):
Looks like there's also a scaling issues with the background image in the credits. To scale w/screen size, this image should also have anchors at corners.

In this case, since the bg is plain black, an alternative way to do it would be to set the background color of the camera & not use an image at all.

(Either way is fine for this, but it does need to scale correctly.)

Commit:
changed credits background

Changed black background from canvas image to main camera background color, to account for screen scaling.
